conf/plants/mlss_conf.xml

```diff
@@ -9,6 +9,8 @@
       <taxonomic_group taxon_name="Eukaryota">
         <!-- But exclude all triticum_aestivum cultivars -->
         <ref_for_taxon name="triticum_aestivum"/>
+        <!-- And exclude all hordeum_vulgare cultivars -->
+        <ref_for_taxon name="hordeum_vulgare"/>
```

Comment:
Hi @JAlvarezJarreta .. If there are not plans to change the Taxonomy IDs of `hordeum_vulgare` or the Barley cultivars in e115, I'm afraid this needs to be configured as an explicit list of excluded genomes as in the original pull request commit.

The current Taxonomy ID of `hordeum_vulgare` is 112509 (Hordeum vulgare subsp. vulgare), while those of the Barley cultivars are 4513 (species Hordeum vulgare).

Reply:
Cheers!